app/build.gradle

```diff
@@ -191,9 +192,29 @@ dependencies {
   implementation project(":domain")
   implementation project(":utility")
 }
+
 // The GeneratedMessageLite implementations of protobufs are depending on protobuf-java
 // instead of protobuf-lite after Android Studio 3.5,
 // The below command stops that from happening: https://github.com/google/tink/issues/282
 configurations {
   all*.exclude module: 'protobuf-java'
 }
+
+
+tasks.withType(Test) {
+  jacoco.includeNoLocationClasses = true
+  jacoco.excludes = ['jdk.internal.*']
+}
+task jacocoTestReport(type: JacocoReport, dependsOn: ['testDebugUnitTest', 'createDebugCoverageReport']) {
+
+  reports {
+    xml.enabled = true
+    html.enabled = true
+  }
+  def mainSrc = "$project.projectDir/src/main/kotlin"
```

Comment:
I think the path here should be `$project.projectDir/src/main/java`. with path as kotlin the coverage report contains no classes.

Reply:
Yes i think so